### PR TITLE
Sustain: Make old article date clearer

### PR DESCRIPTION
**Pull request by JustinPinner**
re: https://trello.com/c/JK4Y1YkE/7-clearer-date-on-old-articles

Sometimes old articles gain new interest (e.g. Tony Hart) and so to prevent them from gathering too much new popularity we need to indicate that they are older pieces.

In the popular container;
screen shot 2015-10-09 at 09 34 50

In an article;
screen shot 2015-10-08 at 12 32 58

In a gallery;
screen shot 2015-10-08 at 11 16 07

### CODE REVIEW

**Comment by Jholder112233** [comment]
I understand the decision to label old articles in the most popular container to avoid confusion, but I really don't understand why we want to add repeated information (the publish date is right above it) to an already crowded meta area (especially on mobile) for the sake of a couple of articles which randomly go viral months after being published.

At the very least, should we consider if galleries really go out of date?

File: common/app/views/support/ContentOldAgeDescriber.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  def message(pubDate: DateTime) = {
+    val warnLimitDays = 7   // config this!
+    if(pubDate.isBefore(DateTime.now().minusDays(warnLimitDays))) {
+      val ageMillis = DateTime.now().getMillis - pubDate.getMillis
+      val years = TimeUnit.MILLISECONDS.toDays(ageMillis) / 365
+      val months = TimeUnit.MILLISECONDS.toDays(ageMillis) / 28
+      val weeks = TimeUnit.MILLISECONDS.toDays(ageMillis) / 7
+      val days = TimeUnit.MILLISECONDS.toDays(ageMillis)
+
+      true match {
```
Comment (gklopper) [inline common/app/views/support/ContentOldAgeDescriber.scala]: never seen anything like this before?

Comment (gklopper) [inline common/app/views/support/ContentOldAgeDescriber.scala]: Did you consider something like http://www.ocpsoft.org/prettytime/

File: common/app/views/support/ContentOldAgeDescriber.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+object ContentOldAgeDescriber extends ContentOldAgeDescriber
+
+class ContentOldAgeDescriber {
+  def apply(content: model.Content): String = {
+    message(content.webPublicationDate)
+  }
+
+  def apply(maybeContent: Option[com.gu.contentapi.client.model.Content]): String = {
+    maybeContent.map{c =>
```
Comment (janua) [inline common/app/views/support/ContentOldAgeDescriber.scala]: `map(apply)` ?

**Comment by paperboyo** [comment]
Not that anybody should care, but I agree with everything @Jholder112233 said (are galleries of happy dead people a thing?!).

There is this thing of excited (or bored?) millions flocking to make a mistake, but will trying to stem the tide with redundant info work?

How to design redundant information well is also a hard question :sadpanda:

**Comment by JustinPinner** [comment]
Some design tweaks incoming. Closing this PR until they're finalised and implemented.

**Comment by JustinPinner** [comment]
Design (styling) changes applied;

screen shot 2015-10-20 at 16 19 44

screen shot 2015-10-20 at 16 21 42

screen shot 2015-10-20 at 16 20 20

**Comment by paperboyo** [comment]
[design] Clock icons inconsistent on white (face either light or dark)? In the first screenshot are clock's hands hollow when intersecting?

**Comment by johnduffell** [comment]
kind of makes me realise how cluttered things are between the main image and the actual start of the content.  I think that whole area needs looking at tbh

**Comment by paperboyo** [comment]
@johnduffell +1

**Comment by johnduffell** [comment]
@JustinPinner you always get the tricky ones...

**Comment by johnduffell** [comment]
how's progress with this one @JustinPinner ?

**Comment by JustinPinner** [comment]
Thought I'd closed this one. We need to limit where we show the message, according to the type of content it applies to. I'll close this until that's done.

**Pull request closed without merging**